src/components/Execution/index.jsx

```diff
@@ -26,10 +22,31 @@ export const Execution = ({ nodes }) => {
   const [webhooks, setWebhooks] = useState([])
   const [singleNode, setSingleNode] = useState(data[0])
 
-  const createWebhooks = useCallback(async () => {
-    const query = async name => {
-      setIsWorking(true)
+  const fetchWebhooks = useCallback(async () => {
+    let { data: webhooks } = await client.collection(TRIGGERS_DOCTYPE).all()
```

Comment:
We avoid using collection when we can, because it bypasses all the store features. Moreover, we have a `useQuery` hooks to do the querying work. And from what I see, we even need `useQueryAll`: https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/hooks/useQueryAll.jsx

Also, I suggest you to group all mango queries into a separate `queries.js` file. We do that for every app, see [here](https://github.com/cozy/cozy-libs/blob/master/packages/cozy-mespapiers-lib/src/helpers/queries.js) for example.

So at the end, that would look like this: 
```
import { triggersQuery } from 'queries'
const { data: webhooks } = useQueryAll(
    triggersQuery.definition,
    triggersQuery.options
  )
```

And the triggersQuery would look like this: 
```
export const triggersQuery = () => ({
  definition: Q(TRIGGERS_DOCTYPE).partialIndex({ type: '@webhook' })
  options: {
    as: `${TRIGGERS_DOCTYPE}/type`,
    fetchPolicy: defaultFetchPolicy
  }
})
```
Note the partial index makes the filter below useless. The sort is also probably unecessary as the docs are already sorted by their id in couchdb